### PR TITLE
Fixed format issue for choroplethstate issue#1815

### PR DESCRIPTION
**Pull request by mounicarachakonda**

### CODE REVIEW

**Comment by jpellock** [comment]
GH-1815

File: src/api/ITooltip.js [hunk 1]
```diff
@@ -109,7 +109,12 @@
         if (opts.value instanceof Date) {
             opts.value = opts.value || "";
         } else {
-            opts.value = this._valueFormatter(opts.value) || "";
+            if(typeof opts.value !== undefined){
+                opts.value = this.fields()[1].transform(opts.value);
```
Comment (jbrundage) [inline src/api/ITooltip.js]: Will this.fields()[1] always return an object?

Comment (mounicarachakonda) [inline src/api/ITooltip.js]: Yes, It always returns an object.

Comment (jpellock) [inline src/api/ITooltip.js]: You should try a test with an empty input dataset.

Comment (mounicarachakonda) [inline src/api/ITooltip.js]: I added this condition only for Tooltip. Tooltip will be called only when there is a data so without data Tooltip will not be called. I have added typeof opts.value condition incase if there is a chance for empty value.

**Comment by GordonSmith** [comment]
The current code uses the "tooltipValueFormat" property to format the data, which may be overkill (having two places for the formatting) - if we are going to use the fields variant (which seems like a good ides), we should remove the tooltipValueFormat property from ITooltip?

**Comment by GordonSmith** [comment]
@mounicarachakonda FYI the general template we use for commit messages looks like this:
```
GH-123 Brief issue summary

Longer description of the changes made.

Fixes GH-123

Signed off by <someuser@company.com>
```
* ```GH-123``` and ```#123``` equate to the same thing.
* The brief description should be limited to about 50 chars (as it ends up in the read me notes).
* The ```fixes GH-123``` will auto close the issue, once this pull request makes it to the master branch.
* The signature is needed to track who  made what change.

**Comment by GordonSmith** [comment]
Closing as there is already a fix for this in 1.14.x

**Pull request closed without merging**